projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -321,3 +288,26 @@ export function outOfStock() {
   cy.get('cx-add-to-cart .quantity').should('contain', 'Out of stock');
   cy.get('cx-add-to-cart cx-add-to-cart button').should('not.exist');
 }
+
+export const cartUser = {
+  user: 'standard',
```

Comment:
```suggestion
  user: 'cart',
```

Use suggestions for typos and simple mistakes.

Reply:
Another cool feature for reviewers in Github is the "Viewed" file feature. Especially useful in multi rounds reviews.